CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # NHS.UK frontend Changelog
 
+## Unreleased
+
+### New features
+
+#### Blue panel variant
+
+There is a new variant of the Panel component with a solid blue background and white text. This can be used as an interruption card. [Pull request #1196](https://github.com/nhsuk/nhsuk-frontend/pull/1196)
```

Comment:
I think we lowercase the component names when inline?

Looks good with the PR link split out for consistency

```suggestion
We've added a new variant of the panel component with a solid blue background and white text. This can be used as an interruption card.
  
This was added in [pull request #1196: Add blue panel (Interruption card) variant](https://github.com/nhsuk/nhsuk-frontend/pull/1196).
```